logging/monolog_email.rst

```diff
@@ -28,6 +28,8 @@ it is broken down.
                     handler:      deduplicated
                 deduplicated:
                     type:    deduplication
+                    # uncomment this line for values other than 60 seconds:
```

Comment:
This comment and the provided `time: 10` example looks a bit confusing to me. What if we explain this option more in detail and don't include the `time: 10` example commented?

```yaml
deduplicated:
    type:    deduplication
    # the time in seconds during which duplicate entries are discarded (default: 60)
    time: 10
    handler: swift
```